[Bug][TaskPlugin] datax job is sucess ,but the instance status is failure.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

datax job is sucess ,but the instance status is failure.

### What you expected to happen

[ERROR] 2022-07-19 18:55:40.745 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[207] - task scheduler failure
java.lang.IllegalThreadStateException: process hasn't exited
at java.lang.UNIXProcess.exitValue(UNIXProcess.java:423)
at org.apache.dolphinscheduler.plugin.task.api.AbstractCommandExecutor.run(AbstractCommandExecutor.java:197)
at org.apache.dolphinscheduler.plugin.task.datax.DataxTask.handle(DataxTask.java:161)
at org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread.run(TaskExecuteThread.java:189)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[60] - FINALIZE_SESSION
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[227] - cancel process: 28379
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[256] - soft kill task:73_74, process id:28379, cmd:sudo -u dolphinscheduler kill 28379
[INFO] 2022-07-19 18:55:41.748 org.apache.dolphinscheduler.service.log.LogClientService:[117] - view log path /opt/apps/dolphinscheduler/logs/6244895070720_6/73/74.log

WaitWriterTime 6.015s | All Task WaitReaderTime 193.856s | Percentage 100.00%
2022-07-19 18:56:11.889 [job-0] INFO JobContainer -
任务启动时刻 : 2022-07-19 18:52:41
任务结束时刻 : 2022-07-19 18:56:11
任务总计耗时 : 210s
任务平均流量 : 55.28KB/s
记录写入速度 : 971rec/s
读出记录总数 : 204104
读写失败总数 : 0

### How to reproduce

[ERROR] 2022-07-19 18:55:40.745 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[207] - task scheduler failure
java.lang.IllegalThreadStateException: process hasn't exited
at java.lang.UNIXProcess.exitValue(UNIXProcess.java:423)
at org.apache.dolphinscheduler.plugin.task.api.AbstractCommandExecutor.run(AbstractCommandExecutor.java:197)
at org.apache.dolphinscheduler.plugin.task.datax.DataxTask.handle(DataxTask.java:161)
at org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread.run(TaskExecuteThread.java:189)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[60] - FINALIZE_SESSION
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[227] - cancel process: 28379
[INFO] 2022-07-19 18:55:40.746 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.datax.DataxTask:[256] - soft kill task:73_74, process id:28379, cmd:sudo -u dolphinscheduler kill 28379
[INFO] 2022-07-19 18:55:41.748 org.apache.dolphinscheduler.service.log.LogClientService:[117] - view log path /opt/apps/dolphinscheduler/logs/6244895070720_6/73/74.log

WaitWriterTime 6.015s | All Task WaitReaderTime 193.856s | Percentage 100.00%
2022-07-19 18:56:11.889 [job-0] INFO JobContainer -
任务启动时刻 : 2022-07-19 18:52:41
任务结束时刻 : 2022-07-19 18:56:11
任务总计耗时 : 210s
任务平均流量 : 55.28KB/s
记录写入速度 : 971rec/s
读出记录总数 : 204104
读写失败总数 : 0

### Anything else

_No response_

### Version

2.0.5

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java
+++ b/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java
@@ -237,8 +237,7 @@ public abstract class AbstractCommandExecutor {
             result.setExitStatusCode(process.exitValue());
 
         } else {
-            logger.error("process has failure , the task timeout configuration value is:{}, ready to kill ...",
-                    result.getExitStatusCode(), taskRequest.getTaskTimeout());
+            logger.error("process has failure, the task timeout configuration value is:{}, ready to kill ...", taskRequest.getTaskTimeout());
             ProcessUtils.kill(taskRequest);
             result.setExitStatusCode(EXIT_CODE_FAILURE);
         }

--- a/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java
+++ b/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java
@@ -237,8 +237,8 @@ public abstract class AbstractCommandExecutor {
             result.setExitStatusCode(process.exitValue());
 
         } else {
-            logger.error("process has failure , exitStatusCode:{}, processExitValue:{}, ready to kill ...",
-                    result.getExitStatusCode(), process.exitValue());
+            logger.error("process has failure , the task timeout configuration value is:{}, ready to kill ...",
+                    result.getExitStatusCode(), taskRequest.getTaskTimeout());
             ProcessUtils.kill(taskRequest);
             result.setExitStatusCode(EXIT_CODE_FAILURE);
         }

